src/test/java/hudson/plugins/active_directory/docker/TheFlintstonesTest.java

```diff
@@ -73,14 +78,16 @@
     public JenkinsRule j = new JenkinsRule();
 
     public final static String AD_DOMAIN = "samdom.example.com";
-    public final static String AD_MANAGER_DN = "CN=Administrator,CN=Users,DC=SAMDOM,DC=EXAMPLE,DC=COM";
+    public final static String AD_MANAGER_DN = "CN=Fred,CN=Users,DC=SAMDOM,DC=EXAMPLE,DC=COM";
     public final static String AD_MANAGER_DN_PASSWORD = "ia4uV1EeKait";
     public final static int MAX_RETRIES = 30;
+    public static String DOCKER_IP;
+    public static int DOCKER_PORT;
```

Comment:
Why `static`? These should be instance variables (and conventionally lowercase).